.github/workflows/release.yml

```diff
@@ -6,52 +6,42 @@ on:
       - 'v*'
 
 jobs:
-  build:
+  ci:
+    uses: ./.github/workflows/ci.yml
+  release:
     name: Create Release
+    needs: ci
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v3.1.0
-        with:
-          fetch-depth: 0
-          ref: main
       - name: Set build variables
-        env:
-          TAG_NAME: ${{ github.ref }}
         run: |
-          export TAG=$(basename $TAG_NAME)
-          echo "TAG=${TAG}"
-          echo "VERSION=${TAG#v}"
-          echo "TAG=${TAG}" >> $GITHUB_ENV
-          echo "VERSION=${TAG#v}" >> $GITHUB_ENV
-      - name: Create Release
-        id: create_release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          tag_name: ${{ github.ref }}
-          release_name: ${{ github.ref }}
-          draft: true
-          prerelease: false
+          echo "VERSION=${GITHUB_REF_NAME#v}" >> $GITHUB_ENV
       - name: Set up Python
         uses: actions/setup-python@v4.2.0
         with:
           python-version: '3.x'
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          python -m pip install --upgrade setuptools
-          pip install build check_manifest setuptools_scm twine
-      - name: Check manifest
-        run: |
-          check-manifest -v
```

Comment:
There's no longer any need to run check-manifest and twine here, because they were already run when the packages were built.